apps/rule-manager/client/src/ts/components/RuleMetadata.tsx

```diff
@@ -8,14 +8,14 @@ import {DraftRule} from "./hooks/useRule";
 import {TagMap} from "./hooks/useTags";
 
 export const RuleMetadata = ({tags, ruleData, partiallyUpdateRuleData}: {
-    ruleData: DraftRule,
+    ruleData: DraftRule | DraftRule[],
     partiallyUpdateRuleData: PartiallyUpdateRuleData,
     tags: TagMap
 }) => {
 
     return <RuleFormSection title="RULE METADATA">
         <LineBreak/>
-        <CategorySelector ruleData={ruleData} partiallyUpdateRuleData={partiallyUpdateRuleData}/>
-        <TagsSelector tags={tags} ruleData={ruleData} partiallyUpdateRuleData={partiallyUpdateRuleData}/>
+        <CategorySelector ruleData={Array.isArray(ruleData) ? ruleData[0]: ruleData} partiallyUpdateRuleData={partiallyUpdateRuleData}/>
```

Comment:
Not very happy with this, please do suggest if there is a cleaner way to implement this